xchange-coinbasepro/src/main/java/org/knowm/xchange/coinbasepro/CoinbaseProAdapters.java

```diff
@@ -100,6 +100,10 @@ protected static Date parseDate(final String rawDate) {
     }
   }
 
+  public static CurrencyPair toCurrencyPair(final String productId) {
+    return new CurrencyPair(productId.replace('-', '/'));
```

Comment:
nit: methods like this are called a _lot_ in high-volume code. This would be more efficient:

```
var split = productId.split('-');
return new CurrencyPair(split[0], split[1]);
```

Reply:
updated